src/main/webapp/PlaceGuideOnList.js

```diff
@@ -0,0 +1,41 @@
+class PlaceGuideOnList {
+  constructor(placeGuideId, place, name, creator, description, audioKey, audioLength, imageKey, createdByCurrentUser) {
```

Comment:
We should add a bookmarkedByCurrentUser field to the constructor too. 
Sorry, I missed it when I created the diagram.